src/web/lib/components/Banner/index.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react';
 import classNames from 'classnames';
 
+import { DOWNLOAD_FIREFOX_URL } from '../../../../lib/constants';
+
 import './index.scss';
 
-export const Banner = ({ addonUrl, bottom = false }) =>
-  <div className ={classNames('banner', { 'banner--bottom': bottom })}>
-    <div className="banner__content">
-      <span>Put the 🔥 in Firefox</span>
-      <a href={addonUrl} className="banner__button">
-        Install Themer
-      </a>
-    </div>
-    {!bottom && <div className="banner__spacer"/>}
-  </div>;
+export const Banner = ({ isFirefox, addonUrl, bottom = false }) => (
+  <div className={classNames('banner', { 'banner--bottom': bottom })}>
```

Comment:
We have `banner--bottom` and `banner__content` here. Could we standardize on a delimiter?

Reply:
Sure? Probably a @johngruen question, since this comes from an earlier commit and I just reformatted here